cumulusci/tasks/salesforce/update_profile.py

```diff
@@ -148,27 +142,20 @@ def _generate_package_xml(self, operation):
 
             package_xml_content = package_xml_content.format(**self.namespace_prefixes)
 
-            if (
-                self.options["include_packaged_objects"]
-                or "package_xml" not in self.options
-            ):
-                # We need to rewrite the package.xml for one or two reasons.
-                # Either we are using packaged-object expansion, or we're using
-                # the built-in admin_profile.xml and need to substitute in
-                # profile API names.
-
-                # Convert to bytes because stored `package.xml`s typically have an encoding declaration,
-                # which `fromstring()` doesn't like.
-                package_xml = metadata_tree.fromstring(
-                    package_xml_content.encode("utf-8")
-                )
+            # We need to rewrite the package.xml for one or two reasons.
+            # Either we are using packaged-object expansion, or we're using
+            # a package.xml and need to substitute in profile API names.
+
+            # Convert to bytes because stored `package.xml`s typically have an encoding declaration,
+            # which `fromstring()` doesn't like.
```

Comment:
This is a tangent from this PR, but we should be sure to specify encoding='utf-8' when opening the file on line 140.